tools/slang-unit-test/unit-test-coverage-tracing-metadata.cpp

```
@@ -144,6 +147,228 @@ SLANG_UNIT_TEST(coverageTracingMetadata)
+                SLANG_CHECK(info.access == slang::SyntheticResourceAccess::ReadWrite);
+                SLANG_CHECK(info.entryPointIndex == -1);
+                SLANG_CHECK(info.debugName != nullptr);
+                SLANG_CHECK(UnownedStringSlice(info.debugName) == toSlice("__slang_coverage"));
+
+                // Descriptor-facing binding info should be available
+                // even when the resource is hidden from ordinary
+                // reflection. For the current CPU source target, the
+                // metadata should also expose the concrete
+                // marshaling location in the generated global params
+                // payload.
+                SLANG_CHECK(info.binding >= 0);
```

Cppcheck (2.20.0)

[style] The function 'Swap' is never used.

(unusedFunction)